modules/core/test/v2/unit/coins/cgld.ts

```diff
@@ -0,0 +1,22 @@
+import { TestBitGo } from '../../../lib/test_bitgo';
+import { Tcgld } from '../../../../src/v2/coins/tcgld';
+import { Cgld } from '../../../../src/v2/coins/cgld';
+
+describe('Celo Gold', function() {
+  let bitgo;
+  let basecoin;
+
+  before(function() {
+    bitgo = new TestBitGo({ env: 'mock' });
+    bitgo.initializeTestVars();
+    basecoin = bitgo.coin('tcgld');
+  });
+
+  it('should instantiate the coin', function() {
+    let localBasecoin = bitgo.coin('tcgld');
+    localBasecoin.should.be.an.instanceof(Tcgld);
```

Comment:
Why does this check pass? Shouldn't the returned instance be an `instanceof AbstractEthLikeCoin` and not `Tcgld`? Am I missing something here?

Reply:
Just checked the tests and these `instanceof` checks are indeed failing (glad I'm not going crazy). Please check

Reply:
Is it still after the latest change? It works for me:
```
▶ mocha --opts test/mocha.opts "test/v2/unit/coins/cgld.ts"       
  
 
(node:17052) DeprecationWarning: Configuration via mocha.opts is DEPRECATED and will be removed from a future version of Mocha. Use RC files or package.json instead.
(node:17052) [DEP0091] DeprecationWarning: crypto.DEFAULT_ENCODING is deprecated.
(node:17052) [DEP0010] DeprecationWarning: crypto.createCredentials is deprecated. Use tls.createSecureContext instead.
(node:17052) [DEP0011] DeprecationWarning: crypto.Credentials is deprecated. Use tls.SecureContext instead.
  Celo Gold
    ✓ should instantiate the coin
```

Reply:
Yep, latest changes fixes it. Thanks